### ISSUE TITLE
Add the ability to change the origin SoC on the origins Edit Waypoint dialog

### ISSUE THREAD

**Issue by aesculus** [aesculus/EVTO-App-Feedback#308]
Users are confused on where to set this value. Many come to the origin to try to set it and are not aware it's in the Edit Segment Details dialog.

I propose leaving it there but also adding it to the origins Edit Waypoint as well.

We should probably review if any other details from the Edit Segment Details dialog should be added as well.

**Comment by EVGrokker** [comment]
I agree with this idea. it should be possible to set the Origin SoC directly by invoking the Edit Waypoint dialog from the Waypoint Inspector.

**Comment by aesculus** [comment]
For now I have added this note to the origin Edit dialog in V1.2 (8)

_Note: Use Edit Segment Details to change the starting SoC and other trip factors._
